crates/lib/src/cli.rs

```diff
@@ -89,6 +89,11 @@ pub(crate) struct UpgradeOpts {
     #[clap(long = "soft-reboot", conflicts_with = "check")]
     pub(crate) soft_reboot: Option<SoftRebootMode>,
 
+    /// Use podman/skopeo to pull image to additionalimagestore, then read from container storage.
+    /// This provides a unified approach that leverages existing container tooling.
+    #[clap(long)]
+    pub(crate) unified: bool,
```

Comment:
I don't think this should be a CLI flag that operates just once; it should be something like `bootc image --set-unified` or so and act persistently.

Also we should support setting this at install time so that it happens from the very start.

Reply:
(hmm my bad I thought we discussed this but I failed to update the issue https://github.com/bootc-dev/bootc/issues/20 or something maybe?)

Reply:
Ahh man yeah, we talked about saving the config on the origin file IIRC, I forgot.